website/content/docs/concepts/password-policies.mdx

```diff
@@ -360,6 +360,39 @@ This policy generates 8 character passwords from the charset `abcde01234` and re
 character from `01234` to be in it, but does not require any characters from `abcde`. The password
 `04031945` may result from this policy, even though no alphabetical characters are in it.
 
+## Default Password Policy
+
+Vault ships with a Default Password Policy and this gets applied to any password that Vault 
+generates where a custom password policy is not explicitly set. The default password policy is 
+defined below.
```

Comment:
```suggestion
--------------------------------------------------------------------------------
Vault ships with a default password policy that applies to any password 
generated by Vault  without an explicit policy assignment. The default
policy requires passwords include:

- 20 characters total
- 1 uppercase character
- 1 lowercase character
- 1 number
- 1 special character
```
Style correction: reserve capitalization for products (not features/functionality), write in the active voice, avoid "this" as a pronoun.